repos/c2sm/packages/makedepf90/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Makedepf90(Package):
+    """ Makedepf90 is a program for automatic creation of
+        Makefile-style dependency lists for Fortran source code."""
+
+    homepage = "https://salsa.debian.org/science-team/makedepf90.git"
+    url = "https://salsa.debian.org/science-team/makedepf90.git"
```

Comment:
Normally URLs point to a tar of the latest version. See https://spack-tutorial.readthedocs.io/en/latest/tutorial_buildsystems.html `url = "https://github.com/llnl/callpath/archive/v1.0.3.tar.gz"`